themes/mongodb/src/js/componentThirdParty.js

```diff
@@ -1,20 +1,12 @@
-const SAMPLE_FACTORS = {
-    'charts': 0.12158,
-    'guides': 0.02644,
-    'stitch': 0.03162,
-    'docs-ruby': 0.47984,
-    'ecosystem': 0.00997,
-    'docs-php-library': 0.04165,
-    'atlas': 0.01176,
-    'compass': 0.01684,
-    'manual': 0.00022,
-    'landing': 0.00806,
-    'mongoid': 0.07576,
-    'mms-cloud': 0.06358,
-    'mms-onprem': 0.01790,
-    'bi-connector': 0.03912,
-    'spark-connector': 0.05562
-};
+const ENABLED_SITES_FOR_DELIGHTED = [
+    'cloud-docs',
+    'datalake',
+    'docs',
+    'guides',
+    'manual',
+    'node',
+    'realm'
+];
```

Comment:
This is a good use case for a JS [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set)!